app/src/js/utils/privileges.js

```diff
@@ -181,3 +193,19 @@ export const questionPrivileges = (privileges) => {
     canDelete: false
   };
 };
+
+/**
+ * given an array of privileges (strings), return an object with keys as the type of privilege and values as an array of actions
+ * 
+ * @example given an array of privileges ['USER_READ', 'USER_CREATE', 'GROUP_READ', 'GROUP_CREATE'], will return { USER: ['READ', 'CREATE'], GROUP: ['READ', 'CREATE'] }
+ */
+export const getPrivilegesByType = (privileges) => {
```

Comment:
Privileges elsewhere are structured as an object of privileges by type. On the requests page, it's an array of strings.

To make use of the privileges utility functions, have to turn the array of strings into an object